### PR TITLE
Added calculation for ingredient nutrient data

### PR DESCRIPTION
**Pull request by Robomarti**

### CODE REVIEW

File: backend/database.js [hunk 1]
```diff
@@ -180,15 +180,40 @@ const doesRestaurantExist = async name => {
...
+    const carbohydrates = nutrientDictionary['carbohydrates'];
+    const protein = nutrientDictionary['protein'];
+    const fat = nutrientDictionary['fat'];
+    const fiber = nutrientDictionary['fiber'];
+    const sugar = nutrientDictionary['sugar'];
+    const salt = nutrientDictionary['salt'];
+    const saturatedFat = nutrientDictionary['saturatedFat'];
+    const energy = nutrientDictionary['energy'];
+
+    const mealMass = nutrientDictionary['mealMass'];
```
Comment (nuclearkittens) [inline backend/database.js:202]: why not just weight :grimacing:

File: e2e-tests/meals.spec.ts [hunk 2]
```diff
@@ -42,6 +42,11 @@ const initTestDB = async () => {
 test.describe('restaurant meal page', () => {
     test.beforeEach(async ({ page }) => {
         await initTestDB();
+        // await page.goto('/login');
+        // await page.fill('input[id="username-input"]', 'test');
+        // await page.fill('input[id="password-input"]', 'Test123!');
+        // await page.locator('#login-button').click();
```
Comment (nuclearkittens) [inline e2e-tests/meals.spec.ts:48]: why r these commented out but not removed :thinking:

File: src/components/ui/Nutriscore.jsx [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+        width: '5%',
+        backgroundColor: 'orange',
+    },
+    rectangleE: {
+        height: '90%',
+        width: '5%',
+        backgroundColor: 'red',
+    },
+});
+
```
Comment (nuclearkittens) [inline src/components/ui/Nutriscore.jsx:259]: sorry but it ugly :crying_cat_face:

**Review by nuclearkittens — DISMISSED** [review]
double check that co2 and nutritional value info is correct (whether per portion or per 100g), otherwise 🤝

**Review by maxxof — APPROVED** [review]
🤝

**Pull request merged**